.github/common/fortran_format_check.py

```diff
@@ -18,7 +18,7 @@
 ]
 
 # Exclude these files from checks
-excludefiles = ["src/Utilities/InputOutput.f90"]  # excluded until refactored
+excludefiles = []  # add excluded files here   # NOTE: as of 1/18/2024 "src/Utilities/InputOutput.f90" no longer needs to be excluded
```

Comment:
Just a quick meta-comment but my feeling is retrospective info like this can generally just go in commit messages. If it's ever necessary to see when a particular line was changed it's just a git command away:

```shell
$ git log -L21,+1:.github/common/fortran_format_check.py --format=oneline --no-patch
5c52165979184b8780628466802c83126184d854 (HEAD -> cleanup_io_docstrings) One last minor change to a comment in .github/common/fortran_format_check.py
e087f43baffa5ed1147c71fa01f22d9ad4202a52 Fix fprettify exclusion to be an empty list
f13f213532e2955e7269fef150907d83ec409386 Some final clean-up, thanks @langevin-usgs and @mjreno
bb62e4ef8892d358d344da73447b194016587949 refactor(ci): convert fortran lint bash script to python (#1150)
```

To see the diff introduced by each commit, omit `--no-patch`

```diff
$ git log -L21,+1:.github/common/fortran_format_check.py --format=oneline
5c52165979184b8780628466802c83126184d854 (HEAD -> cleanup_io_docstrings) One last minor change to a comment in .github/common/fortran_format_check.py

diff --git a/.github/common/fortran_format_check.py b/.github/common/fortran_format_check.py
--- a/.github/common/fortran_format_check.py
+++ b/.github/common/fortran_format_check.py
@@ -21,1 +21,1 @@
-excludefiles = []  # excluded until refactored # NOTE: as of 1/18/2024 "src/Utilities/InputOutput.f90" no longer needs to be excluded
+excludefiles = []  # add excluded files here   # NOTE: as of 1/18/2024 "src/Utilities/InputOutput.f90" no longer needs to be excluded
e087f43baffa5ed1147c71fa01f22d9ad4202a52 Fix fprettify exclusion to be an empty list

diff --git a/.github/common/fortran_format_check.py b/.github/common/fortran_format_check.py
--- a/.github/common/fortran_format_check.py
+++ b/.github/common/fortran_format_check.py
@@ -21,2 +21,1 @@
-# (the following line commented out on 1/18/2024)
-# excludefiles = ["src/Utilities/InputOutput.f90"]  # excluded until refactored
+excludefiles = []  # excluded until refactored # NOTE: as of 1/18/2024 "src/Utilities/InputOutput.f90" no longer needs to be excluded
f13f213532e2955e7269fef150907d83ec409386 Some final clean-up, thanks @langevin-usgs and @mjreno

diff --git a/.github/common/fortran_format_check.py b/.github/common/fortran_format_check.py
--- a/.github/common/fortran_format_check.py
+++ b/.github/common/fortran_format_check.py
@@ -21,1 +21,2 @@
-excludefiles = ["src/Utilities/InputOutput.f90"]  # excluded until refactored
+# (the following line commented out on 1/18/2024)
+# excludefiles = ["src/Utilities/InputOutput.f90"]  # excluded until refactored
bb62e4ef8892d358d344da73447b194016587949 refactor(ci): convert fortran lint bash script to python (#1150)

diff --git a/.github/common/fortran_format_check.py b/.github/common/fortran_format_check.py
--- /dev/null
+++ b/.github/common/fortran_format_check.py
@@ -0,0 +21,1 @@
+excludefiles = ["src/Utilities/InputOutput.f90"]  # excluded until refactored
```

Reply:
very helpful, @wpbonelli

Reply:
Have removed the comment in question with 50cc179